app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+// -- Headers ----------------------------------------------------------------------
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Roobert-Bold', 'Times';
+    font-style: normal;
+    font-weight: bold;
+    color: $blue_primary;
+}
+
+// ---------------------------------------------------------------------------------
+
+
+// -- Paragraphs -------------------------------------------------------------------
+p{
+    font-family: 'Roobert-Regular', sans-serif!important;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 23px;
+    color: $text_secondary;
+    &.callout{
+        background-color: $light_grey;
+        border-radius: $border_radius;
+        padding: 10px;
```

Comment:
Can we use the `$baseline-unit` variable inherited from Yeast here (at least for the vertical padding). This is generally preferable to maintain "vertical rhythm".